docs/Masters/IELTS/index.md

```diff
@@ -0,0 +1 @@
+# IELTS Academic
```

Comment:
Would suggest not to keep a blank page, but if it's there to encourage a contribution or as something on a nearby roadmap then sure.

Reply:
> encourage a contribution

Yes